doc/guides/beyond-GSG.rst

```diff
@@ -0,0 +1,277 @@
+.. _beyond-gsg:
+
+Beyond the Getting Started Guide
```

Comment:
I don't think this should be under User Guides. This should instead be inside the GSG hierarchy in the ToC in my opinion.